### ISSUE TITLE
Enchants

### ISSUE THREAD

**Issue by LostPoE** [Essyer/PoETiS#12]
Can you add enchants to the modifiers? For example, if a ring or amulet has X anointment enchant?

Thanks

**Comment by Essyer** [comment]
@LostPoE I could try, but do you have an idea how it could be presented? Right now different colors represent number of "good" suffixes and dashed line means there is no more space for new affixes.

**Comment by LostPoE** [comment]
Could add a star icon that represents an enchant. Thanks for the quick response!

**Comment by Essyer** [comment]
Good idea, will add it to my TODO list. For anointments I could add several stars depending on what oils are part of it. I'm not sure about lab enchants though, the value of enchant may differ from league to league, maybe I could use poe.ninja API.

**Comment by LostPoE** [comment]
Any anointment that has Golden Oil is worth keeping.

**Comment by LostPoE** [comment]
http://www.vhpg.com/anointing-rings/ You can filter by oil type in the search bar at the top right. Try searching for gold.

**Comment by LostPoE** [comment]
Are the searches able to find these anointments yet?

**Comment by Essyer** [comment]
Not yet, sorry, didn't have much time lately to work on it. I will let you know as soon as it's added.

**Comment by LostPoE** [comment]
Thanks!
